index.html

```diff
@@ -57,10 +57,10 @@ <h3 class="profession">I'am a Software Developer</h3>
         <li class="flex1">Ruby</li>
       </ul>
     </div>
-    <div class="button1"><button id="button1">See Project</button></div>
+    <div class="button1"><button id="button1" type="button">See Project</button>
```

Comment:
- [ ] Good job designing the button. Kindly add the various button interactions to all your buttons so as to improve the user interface 👍 
- Button interactions:
![Annotation 2022-03-18 061338](https://user-images.githubusercontent.com/85113567/158941697-36aa808a-1478-475f-aafa-a4c7b48aab29.jpg)